packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+This page provides a detailed description of the various key prototypes found in the Celo protocol, as well as small examples of how to designate key types in the Celo proof-of-stake smart contracts.
+
+## Undesignated
+
+By default, any private key generated in the Celo protocol does not have a specific role in proof-of-stake. These keys can be used to sign and send transactions that are unrelated to Celo's proof-of-stake protocol, including but by no means limited to transferring Celo Gold, transferring Celo Dollars, exchanging Celo Gold for Celo Dollars, etc.
+
+Undesignated keys can be designated as Locked Gold Accounts or authorized as signer keys on behalf of a Locked Gold Account by sending special transactions using `celocli`.
+
+Note that [ReleaseGold](../../celo-codebase/protocol/release-gold) beneficiary keys are by default Undesignated with respect to proof-of-stake, and that the `ReleaseGold` contract address is what ultimately gets designated as a Locked Gold Account.
+
+
+## Locked Gold Accounts
+
+Locked Gold Account keys have the highest level of privileges in the Celo protocol. These keys can be used to lock and unlock cGLD in order to be used in proof-of-stake. Furthermore, Locked Gold Account keys can be used to authorize other keys to sign transactions and messages on behalf of the Locked Gold Account.
+
+In *most* cases, the Locked Gold Account key has all the privileges as any authorized keys. For example, if a voter signing key is authorized, a user can place votes on behalf of the Locked Gold Account with both the authorized vote signing key *and* the Locked Gold Account key.
+
+Because of the significant priveleges afforded to the Locked Gold Account key, it is best to store this key securely and access it as infrequently as is possible. Authorizing other signer keys is one way to minimize how frequently you need to access your Locked Gold Account key. The Locked Gold Account key will only be used to send transactions and **can be stored on a Ledger hardware wallet.**
+
+### Creating a Locked Gold Account
+An undesignated key may be designated as a Locked Gold Account key by running the following command:
+
+```shell
+# Designate the key as a Locked Gold Account
+celocli account:create --from $ADDRESS_TO_DESIGNATE --useLedger
+
+# Confirm the key was designated as a Locked Gold Account
+celocli account:show $ADDRESS_TO_DESIGNATE
+```
+
+## Authorized Vote Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a vote signing key. Authorized vote signing keys can vote in validator elections and on-chain governance on behalf of the Locked Gold Account.
+
+Note that the vote signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+Authorized vote signing keys will only be used to send transactions and **can be stored on a Ledger hardware wallet**.
```

Comment:
worth specifying here that they cannot be used to move funds?

Reply:
"can only be used to send voting txs"